### PR TITLE
translating HOWTO.md to Dutch

### PR DESCRIPTION
**Pull request by JenniX3**
## What does this PR do?
Improve repo

### Description

Translated HOWTO to Dutch

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by SethFalco — APPROVED** [review]

**Comment by eshellman** [comment]
I would separate the translation and the links to it into two separate PRs. The links generate the conflicts, and we can do a cleanup PR after things have settled a bit.

File: HOWTO-nl.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Lees dit in andere talen [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md).
+
+Welkom bij Free-Programming-Books! We verwelkomen nieuwe bijdragers; zelfs degenen die hun allereerste pull-verzoeken doen. Als je een van hen bent, zijn hier enkele bronnen die je kunnen helpen:
+
+* [Over pull-verzoeken](https://help.github.com/articles/about-pull-requests/)
+* [Een pull-verzoek maken](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hallo Wereld](https://guides.github.com/activities/hello-world/)
+* [Youtube - Github-zelfstudie voor beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Hoe een GitHub-repo te forken en een pull-verzoek in te dienen](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
```
Comment (davorpa) [inline HOWTO-nl.md:10]: Take into account to add a note according to this links are in english. Translate `in english`

Example:
```markdown
* [Over pull-verzoeken](https://help.github.com/articles/about-pull-requests/) *(in english)*
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: HOWTO-nl.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+Aarzel niet om vragen te stellen; elke bijdrager begon met een eerste PR. Je zou onze duizendste kunnen zijn!
+
+Zelfs als je een ervaren open source-bijdrager bent, zijn er dingen die je kunnen laten struikelen. Nadat je je PR hebt ingediend, voert GitHub Actions een linter uit, waarbij vaak kleine problemen met spatiëring of alfabetisering worden gevonden. Als je een groene knop krijgt, is alles klaar voor beoordeling, maar als dat niet het geval is, klik je op "Details" onder het vinkje dat niet heeft kunnen achterhalen wat de linter niet leuk vond. Los het probleem op en voeg een commit toe aan je PR.
+
+Tot slot, als je niet zeker weet of de bron die je wilt toevoegen geschikt is voor gratis programmeerboeken, lees dan de richtlijnen in [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (davorpa) [inline HOWTO-nl.md:17]: Help us with CONTRIBUTING translation if you want

**Review by davorpa — CHANGES_REQUESTED** [review]
Users should be warned about the language used in external links content (https://github.com/EbookFoundation/free-programming-books/pull/5831/files#r722720638).

File: HOWTO-nl.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Lees dit in andere talen [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Tiếng Việt](HOWTO-vi.md), [عربي](HOWTO-ar.md).
```
Comment (davorpa) [inline HOWTO-nl.md:1]:
```suggestion
Lees dit in andere talen: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Tiếng Việt](HOWTO-vi.md), [عربي](HOWTO-ar.md).
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**